moabb/datasets/ssvep_exo.py

```diff
@@ -2,24 +2,27 @@
 
 from mne.io import Raw
 
+from moabb.utils import depreciated_alias
+
 from . import download as dl
 from .base import BaseDataset
 
 
 SSVEPEXO_URL = "https://zenodo.org/record/2392979/files/"
 
 
-class SSVEPExo(BaseDataset):
+@depreciated_alias("SSVEPExo", "0.7")
+class Exoskeleton_SSVEP(BaseDataset):
```

Comment:
Why keep SSVEP in the name, where the other do not have it? (`Nakanishi2015`, `Wang2016`, `MAMEM1`, ...)
To keep in line with the name scheme, we could use `Kalunga2016` instead of `Exoskeleton_SSVEP`